README.md

```diff
@@ -144,7 +144,7 @@ After taking the request through the POST API, the app will be generated and ema
 
 ### Screenshots of the app generator web page
 **Server's Landing Page** <br>
```

Comment:
Please change this to something else, other than Server's landing page. Something like See it in action etc.